doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -81,6 +81,16 @@ Do not enable the controller-based startup strategy in an environment with a hig
 ====
 +
 . Optional: If you want to use the OpenShift CA bundle as the trust store for HTTPS communication, select the *Use OpenShift CA Bundle* box.
+. Optional: If you want to use a secret that contains the credentials for the Admin user, complete the following tasks:
+.. From the *Admin user configuration* list, select `Secret configuration`.
+.. Under *OpenShift admin credentials secret*, in the *Secret* field, enter the name of the secret you want to use. If the *Secret* field is left blank, the default secret from `kie-admin-credentials` is used.
+.. Under *OpenShift admin credentials secret*, in the *Username* field, enter the username for the admin user. If the *Username* field is left blank, the default username from `kie-admin-credentials` is used.
```

Comment:
Under *OpenShift admin credentials secret*, in the *Username* field, enter the username for the admin user to use in the secret created

Reply:
Just to clarify, these fields are input variables for the creation of a secret, not input variables to match an existing secret?

Reply:
Done